clang/lib/CAS/CASOptions.cpp

```diff
@@ -17,13 +17,6 @@
 using namespace clang;
 using namespace llvm::cas;
 
-void clang::getClangDefaultCachePath(SmallVectorImpl<char> &Path) {
```

Comment:
Could you also remove the declaration from `CASOptions.h` header?

Reply:
Yes of course!

Reply:
Sorry I missed that